src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.java

```diff
@@ -541,6 +540,21 @@ public void getOffenceHistoryIncludeOffenderWithoutConviction() {
         assertThatJsonFileAndStatus(response, 200, "offender_offence_history_A12234AB_include_non_convictions.json");
     }
 
+    @Test
+    public void getOffenceHistoryForBookings() {
+        final var token = validToken(List.of("ROLE_VIEW_PRISONER_DATA"));
```

Comment:
For future changes prefer to use webTestClient rather than testRestTemplate and prefer JSON asserts in response rather than assertThatJsonFileAndStatus